config/capsule.yaml

```diff
@@ -8,13 +8,133 @@
   :answer_file: "./config/capsule-answers.yaml"
   :installer_dir: "."
   :default_values_dir: "/tmp"
-  :module_dirs: ["/usr/share/foreman/modules", "./modules"]
+  :module_dirs: ["./_build/modules", "../katello-installer/modules"]
   :parser_cache_path:
     - /usr/share/foreman-installer/parser_cache/foreman.yaml
     - /usr/share/katello-installer-base/parser_cache/katello.yaml
+    - /usr/share/katello-installer-base/parser_cache/capsule.yaml
   :hiera_config: /usr/share/foreman-installer/config/foreman-hiera.conf
   :hook_dirs: ["./hooks"]
   :colors: true
   :order:
     - certs
     - capsule
+  :mapping:
+    :foreman::cli:
+      :dir_name: foreman
+      :manifest_name: cli
+      :params_name: cli/params
+    :foreman::plugin::bootdisk:
```

Comment:
Hm, these shouldn't be here in the capsule scenario, these are all foreman plugins. What added them?